musicbot/bot.py

```diff
@@ -133,6 +133,7 @@ async def _doBotInit(self):
                     aiosession=self.session,
                     loop=self.loop,
                 )
+                await self.spotify.get_token()
```

Comment:
This was missing, so it would just not even try to get the token, then complain it didn't have one

Reply:
Good catch!